app/editor/src/components/form/datepicker/SelectDate.tsx

```diff
@@ -39,7 +42,7 @@ export interface IDatePickerProps extends ISelectDateProps, ReactDatePickerProps
  */
 export const SelectDate: React.FC<IDatePickerProps> = ({
   id,
-  name,
+  name = 'date',
```

Comment:
Default the value isn't ideal, but for now it resolves the issue of extracting the error value.